resources/fervent-ui/Components/dialogue.css

```diff
@@ -0,0 +1,97 @@
+.dialogue-container {
+    display:flex;
+    flex-flow: row wrap;
+    gap: 3rem;
+    margin: 3rem;
+    align-items: center;
+}
+
+.dlg-box {
+    /* position: absolute;
+    top:50%;
+    left:50%;
+    transform: translate(-50%, -50%); */
```

Comment:
you can remove unwanted comments

Reply:
Ahh, I missed that. Thank you so much for reviewing!